src/app/dataset-view/additional-components/metadata-component/components/form-components/select-date-format-field/select-date-format-field.component.ts

```diff
@@ -48,6 +48,10 @@ export class SelectDateFormatFieldComponent extends BaseField implements OnInit
                     this.currentSource = EventTimeSourceKind.FROM_METADATA;
                     this.eventTimeGroup.removeControl(this.PATTERN_NAME_CONTROL);
                     this.eventTimeGroup.removeControl(this.TIMESTAMP_FORMAT_NAME_CONTROL);
+                } else if (kind === this.eventTimeSourceKind.FROM_SYSTEM_TIME) {
```

Comment:
Code duplication. Just compare if `kind !== EventTimeSourceKind::FROM_PATH` and do removals once

Reply:
Also, why don't we use type directly? 
The field `eventTimeSourceKind` is only for UI template, btw, it should be named identically to the type.